.migration-backup/ui-components/app/(management)/index.tsx

```
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import { useRouter } from 'expo-router';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)